runtime/doc/vim_diff.txt

```diff
@@ -72,6 +72,12 @@ the differences.
 
 - |g:vimsyn_embed| defaults to "l" to enable Lua highlighting
 
+==============================================================================
+3. Nvim default Mappings                              *nvim-default-mappings*
```

Comment:
don't need to mention "Nvim" in a section of a doc where every line in the doc is relevant to Nvim.
```suggestion
3. Default Mappings                              *nvim-default-mappings*
```